### ISSUE TITLE
For GB section, make simulation with hyperparameters to see which affects more overfitting

### ISSUE THREAD

**Issue by cimentadaj** [cimentadaj/ml_socsci#10]
For example, how does the RMSE change with increasing node size and how that pairs up with the tree depth and learning rate.
